src/currencyexchange/manifest.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Do we care for cloud foundry?

Reply:
@Grant-Steinfeld I thought this was supposed to be deployed to k8s?